site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -475,157 +475,18 @@ With the environment created, we can now create the application package for the
-
-5. Click **Browse**, navigate to the local **sfguide-native-apps-chairlift/app/src/ui** folder on your computer, select all files, and upload them. The folder structure within this stage should now reflect the folder structure in your local **app/** directory.
-
-It is important to make sure the directory structures match so that any references to objects needed by the app do not break. See the screenshots below for reference.
-
-![Upload modal with ui/ in field](assets/upload.png)
-
-![Final directory structure](assets/structure.png)
-
-<!-- ------------------------ -->
-## Create the first version of the app
-Duration: 2
-
-Let's review what we've covered so far:
+Open a new terminal in the root of the repository and execute the following command:
 
-* Created necessary account roles and objects, and granted proper privileges
-
-* Created the application package and uploaded the app's source code to the application package
-
-Next, you'll create the first version of the app. Run the following SQL in a worksheet:
-
-```sql
-use role chairlift_provider;
-alter application package chairlift_pkg add version develop using '@chairlift_pkg.code.source';
-```
-
-This statement will create the first (new) version of the native app using the source code files that you uploaded earlier.
-
-> aside positive
-> 
->  **PATCH VERSIONS** Do not run the SQL statement below. It is included here to demonstrate how you can add a patch version of a native app.
-
-In the scenario where you update the source code for the app to roll out a fix (i.e., fixing a bug), you could add the updated source as a patch to the native app using the following SQL statement:
-
-```sql
-alter application package chairlift_pkg add patch for version develop using '@chairlift_pkg.code.source';
-```
-
-This SQL command returns the new patch number, which will be used when installing the application as the consumer.
-
-<!-- ------------------------ -->
-## Install the application
-Duration: 3
-
-Now that the source code has been uploaded into the application package, we can now install the application. To install the application in the same account, the provider role needs to grant installation and development permissions to the consumer role. 
-
-To do this, execute the following SQL statement using role **chairlift_provider**:
-
-```sql
-use role chairlift_provider;
-grant install, develop on application package chairlift_pkg to role chairlift_admin;
+```bash
+snow app run
 ```
 
-Next, execute the **consumer/install-app.sql** script. This script installs the application in the account, and grants appropriate privileges to the **chairlift_viewer** role. Note that the version and/or patch values may need to be updated to install the application using a different version or patch.
+This command will upload source files and create the application package and install the application instance automatically. 
```

Comment:
```suggestion
This command will upload source files, create the application package, and install the application object automatically. When you run it again, it will perform the minimum steps necessary to ensure the application is up-to-date with your local copy. 
```